src/components/Card/Card.js

```diff
@@ -24,10 +24,15 @@ function handleAnalytic(channelName) {
   }
 }
 
-const tagsList = tags =>
+const tagsList = (tags, handleTagClick) =>
   tags.map((tag, index) => {
     return (
-      <div className="tag" key={index}>
+      <div
```

Comment:
https://github.com/Coding-Coach/find-a-mentor/pull/193/files#r270911120

Reply:
I changed it to use a `button` element. I just had to set the border to 0 to remove the default browser styling for buttons. It now looks the same as before.